carousel.js

```diff
@@ -24,12 +24,12 @@ const carouselImg = [
 ];
 
 for (let i = 0; i < carouselImg.length; i++) {
-    const img_creater = document.createElement("img")
-    img_creater.setAttribute("id", carouselImg[i].id);
-    img_creater.setAttribute("class", carouselImg[i].class);
-    img_creater.setAttribute("src", carouselImg[i].srs);
-    img_creater.setAttribute("alt", carouselImg[i].alt);
-    carouselImagewrapper.append(img_creater);
+    const threeImages = document.createElement("img")
```

Comment:
1. How work `for` loop?
2. How work `document.createElement("img")`?
3. `document.createElement("img")` creating three images? 
4. Why the name is `threeImages`?